client/main.cpp

```diff
@@ -406,6 +411,81 @@ int boinc_main_loop() {
     return finalize();
 }
 
+//
+//
+//
+int connect_sock () {
+
+    int portno = 31416;
```

Comment:
Maybe this should be a parameter somewhere in cc_config.xml rather than just hardcoded number?